pkg/cli/complete_part2.go

```
@@ -0,0 +1,79 @@
+		Phase:           phase,
+	})
+	if err != nil {
+		return err
+	}
+	if !changed {
+		return nil
+	}
+	return os.WriteFile(specPath, []byte(updated), 0644)
+}
+
+func isV2Feature(feat *feature.Feature) bool {
```

golangci-lint (2.12.2)

[error] func isV2Feature is unused

(unused)